src/btree/bt_vrfy.c

```diff
@@ -439,8 +439,10 @@ __verify_tree(
      * utilizing the regular tree walk function. Check for potential pages to pre-fetch here as
      * well.
      */
-    if (__wt_session_prefetch_check(session, ref))
-        WT_RET(__wt_btree_prefetch(session, ref));
+    if (__wt_session_prefetch_check(session, ref)) {
+        ret = __wt_btree_prefetch(session, ref);
+        WT_PREFETCH_ASSERT(session, ret != WT_ERROR, block_prefetch_failed_start);
```

Comment:
If it's acceptable for prefetch to fail to start, we should probably set `ret = 0` after the assertion (and add a comment).

Reply:
Done ! [c101518](https://github.com/wiredtiger/wiredtiger/pull/9949/commits/c101518d8256eeacb3a2f77f9de61da5ef649f6c)